jina/types/document/__init__.py

```diff
@@ -402,7 +401,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
```

Comment:
eventually we will remove the `context manager` concept from `document` right? or should we at least do some check like wether `id` is not None or not empty?

Reply:
It cannot be None or empty at the point. It will always be populated. This is just for backward compatibility and for "niceness".

Reply:
And out of a sudden, it is not `pass` anymore. Reasoning: It makes sense to update the `content_hash` after `Document` is completed. Otherwise, the user would have to call the `update_content_hash` manually.